main.go

```diff
@@ -197,16 +200,25 @@ func main() {
 		// ``MINIO_PROFILER`` supported options are [cpu, mem, block].
 		switch os.Getenv("MINIO_PROFILER") {
```

Comment:
Move this into a setProfilerFromEnv() function and return the profile to avoid the clutter inside app.Before